### PR TITLE
configuration of cypress, sample test to check if works

### PR DESCRIPTION
**Pull request by eCzerniak**
Adding cypress with sample test.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,7 @@ yarn-error.log*
 
 # localcache
```
Comment (michalina-graczyk) [inline .gitignore:37]: Do we need these comments?

File: .gitignore [hunk 2]
```diff
@@ -36,3 +36,7 @@ yarn-error.log*
...
 .eslintcache
+
+#cypress
```
Comment (michalina-graczyk) [inline .gitignore:40]: same here ^

File: .gitignore (hunk 1 above)
Comment (eCzerniak) [reply .gitignore:37]: this concept has already been addressed, so this question should be addressed to @krzysztofwolski or @zaiste

**Review by zaiste — CHANGES_REQUESTED** [review]
@eCzerniak Great initiative! Would it be possible to default to TypeScript when writing those tests? This could simplify the management of this code base

**Review by michalina-graczyk — APPROVED** [review]

**Review by zaiste — APPROVED** [review]

**Pull request merged**